<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiningFrame</class>
 <widget class="QFrame" name="MiningFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="m_miningSplitter">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>State</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="2">
        <widget class="QPushButton" name="m_stopSolo">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stop mining</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">m_soloButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Mining status:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="m_startSolo">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Start mining</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">m_soloButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Speed, kH/s:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="m_soloLabel">
         <property name="text">
          <string>Stopped</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Difficulty:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLCDNumber" name="m_hashratelcdNumber">
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="m_difficulty">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Settings</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>CPU cores</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QDial" name="m_cpuDial">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="tabletTracking">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>99</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
           <property name="notchTarget">
            <double>4.000000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSpinBox" name="m_cpuCoresSpin">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QCustomPlot" name="m_hashRateChart" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>300</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="m_minerLog">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_soloButtonGroup</sender>
   <signal>buttonClicked(QAbstractButton*)</signal>
   <receiver>MiningFrame</receiver>
   <slot>startStopSoloClicked(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_startSolo</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_startSolo</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_stopSolo</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_stopSolo</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cpuDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_cpuCoresSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cpuCoresSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_cpuDial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startStopClicked(QAbstractButton*)</slot>
  <slot>startStopSoloClicked(QAbstractButton*)</slot>
  <slot>stakeAmountChanged(int)</slot>
  <slot>stakeTermChanged(int)</slot>
  <slot>stakeMixinChanged(int)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="m_soloButtonGroup"/>
 </buttongroups>
</ui>
